import.py

```diff
@@ -52,16 +52,16 @@ def add_transaction(book, item, currency):
     s1 = Split(book)
     s1.SetParent(tx)
     s1.SetAccount(acc)
-    amount = int(item.split_amount.replace('.', ''))
-    s1.SetValue(GncNumeric(amount, 100))
-    s1.SetAmount(GncNumeric(amount, 100))
+    amount = int(float(item.split_amount) * currency.get_fraction())
```

Comment:
Using a "float" with money values is always a bad idea (see e.g. https://vladzloteanu.wordpress.com/2010/01/11/why-you-shouldnt-use-float-for-currency-floating-point-issues-explained-for-ruby-and-ror/)